### PR TITLE
Add information for uploadToItch.md

### PR DESCRIPTION
**Pull request by vaasut**
## Description

This page was a stub. I added some information on how to upload a game to [itch.io](itch.io)

### CODE REVIEW

File: docs/docs/faq/uploadToItch.md [hunk 1]
```diff
@@ -1,9 +1,21 @@
...
+* An itch.io account 
+* Your bitsy game downloaded. If your bitsy game requires more than the html file, put all relevant files into a folder. Then rename the main html file to index.html, and zip the folder.
+* A cover image
+* Some screenshots of your game
+
+1. Sign in to your [itch.io](https://itch.io) account.
+   
+2. Go to your [dashboard](https://itch.io/dashboard) and click on [Create New Project](https://itch.io/game/new).
+
+3. Fill out the fields for your game, and upload your game file, cover image, and screenshots. Bitsy games are executable in the web browser, but you can also set it to be downloadable if you prefer. You can specify the viewport dimensions; 512px x 512px is a good option for many bitsy games.
```
Comment (seleb) [inline docs/docs/faq/uploadToItch.md:14]:
```suggestion
3. Fill out the fields for your game, and upload your game file, cover image, and screenshots. Bitsy games are executable in the web browser, but you can also set it to be downloadable if you prefer. You can specify the viewport dimensions; 512x512px is a good option for many bitsy games. 
```

File: docs/docs/faq/uploadToItch.md [hunk 2]
```diff
@@ -1,9 +1,21 @@
...
+* A cover image
+* Some screenshots of your game
+
+1. Sign in to your [itch.io](https://itch.io) account.
+   
+2. Go to your [dashboard](https://itch.io/dashboard) and click on [Create New Project](https://itch.io/game/new).
+
+3. Fill out the fields for your game, and upload your game file, cover image, and screenshots. Bitsy games are executable in the web browser, but you can also set it to be downloadable if you prefer. You can specify the viewport dimensions; 512px x 512px is a good option for many bitsy games. 
+
+4. Save the game as a draft and view the itch.io page for your game You can make any style changes using the edit theme button. If you want to edit the html for the game page, you can do this back on your project dashboard.
```
Comment (seleb) [inline docs/docs/faq/uploadToItch.md:16]:
```suggestion
4. Save the game as a draft and view the itch.io page for your game. You can make any style changes using the edit theme button. If you want to edit the html for the game page, you can do this back on your project dashboard.
```

**Review by seleb — DISMISSED** [review]
couple little formatting fixes but otherwise looks good to me!

**Review by seleb — APPROVED** [review]

**Comment by le-doux** [comment]
looks like a great addition - thank you @vaasut! thanks for the review and edits @seleb!

**Pull request merged**